src/libs/installer/eventlogger.cpp

```diff
@@ -101,6 +101,18 @@ void EventLogger::sendAllocatedEvent(google::protobuf::Message* payload)
     qDebug() << "framework | EventLogger::sendAllocatedEvent | gateway url =" << s_gatewayUrl;
     QByteArray byteEvent = toJsonByteArray(&event);
     m_httpThreadController->postTelemetry(byteEvent, s_gatewayUrl);
+
+    // Also send the telemetry to our provided endpoint
+    if (QInstaller::useProvidedTelemetryEndpoint())
+    {
+        QString customEndpoint = QInstaller::getProvidedTelemetryEndpoint();
+        qDebug() << "framework | EventLogger::sendAllocatedEvent | Also sending to user provided endpoint =" << customEndpoint;
+        if (customEndpoint == s_gatewayUrl) {
```

Comment:
Yes, I completely agree. But since we don't have any standards (that I'm aware of) I simply followed the format of the code around. But if we prefer brackets having their own lines, then I can easily update the PR